airflow/configuration.py

```diff
@@ -280,9 +280,7 @@ def run_command(command):
 smtp_host = localhost
 smtp_starttls = True
 smtp_ssl = False
-smtp_user = airflow
```

Comment:
Do you think it will be better to group and comment these two lines with a note to uncomment these by changing them to appropriate values if authentication is needed? This might help users in knowing what values to override if they want authentication.